github-electron/github-electron.d.ts

```diff
@@ -401,7 +401,7 @@ declare namespace Electron {
 		 * Note: This will not affect process.argv, and is mainly used by developers
 		 * to control some low-level Chromium behaviors.
 		 */
-		appendSwitch(_switch: string, value?: string|number): void;
+		appendSwitch(_switch: string, value?: string): void;
```

Comment:
Looking into the actual implementation, the value argument is either a string or it's ignored

```
  std::string value;
  if (args->GetNext(&value))
    command_line->AppendSwitchASCII(switch_string, value);
  else
    command_line->AppendSwitch(switch_string);
```